users/views.py

```diff
@@ -1,6 +1,10 @@
 # -*- coding: utf-8 -*-
 from __future__ import unicode_literals
 
-from django.shortcuts import render
+from django.views import generic
 
-# Create your views here.
+from users.models import User
+
+
+class CurrentUserView(generic.ListView):
```

Comment:
I would prefer to have this (and other non-API-views) as method-based views and not class-based views. If you disagree we could of course have a discussion about it.

Reply:
I agree. Class-based views has become a habit for me, but after looking at function-based views, they are much more suited for this project